README.md

```diff
@@ -54,14 +54,14 @@ Please refer to the following instructions to setup Oppiabot for the first time
     * **Webhook Secret**: `development`
     * **Permissions & events** The following permissions and events must be subscribed.If, for example, you only enable issue events, you will not be able to listen on pull request webhooks with the bot.
 
-    <p align="center">
-    <img src="images/repo1-permissions.jpg" height="600" width="400">
-    <img src="images/repo2-permissions.jpg" height="600" width="400">
-    <img src="images/org-permissions.jpg" height="600" width="400">
-    <img src="images/user-permissions.jpg" height="600" width="400">
-    <img src="images/events1.jpg" height="600" width="400">
-    <img src="images/events2.jpg" height="600" width="400">
-</p>
+     <p align="center">
+    <img src="images/docs/repo1-permissions.jpg" height="600" width="450">
+    <img src="images/docs/repo2-permissions.jpg" height="600" width="450">
+    <img src="images/docs/org-permissions.jpg" height="600" width="450">
+    <img src="images/docs/user-permissions.jpg" height="600" width="450">
+    <img src="images/docs/events1.jpg" height="600" width="450">
+    <img src="images/docs/events2.jpg" height="600" width="450">
+    </p>
```

Comment:
```suggestion
<p align="center">
    <img src="images/docs/repo1-permissions.jpg" height="600" width="450">
    <img src="images/docs/repo2-permissions.jpg" height="600" width="450">
    <img src="images/docs/org-permissions.jpg" height="600" width="450">
    <img src="images/docs/user-permissions.jpg" height="600" width="450">
    <img src="images/docs/events1.jpg" height="600" width="450">
    <img src="images/docs/events2.jpg" height="600" width="450">
</p>
```

The images still look stretched, you should try to preserve the aspect ration of these iamges when you're resizing them.